app/assets/stylesheets/module-investments.scss

```diff
@@ -1,3 +1,6 @@
+@import "css-conf";
+@import "mixins";
```

Comment:
Important point here. We must import, at the top of the module CSS, all the required CSS for build the module. 
Note there are only two files imported here, that's because we are importing yet the main bundle, who contains all remaining styles.

The second step of this migration is about include here EVERY SINGLE FILE we need to show the module properly. That is, for instance, we're using some CSS classes belong to the `comp-tables.scss`, then we will include here such file also.

An exception would be those CSS files related with javascript, i.e. vue-components or default styles from javascript features, like read-more, horizontal-slider... who will be imported together the JS, but don't in this point.